components/contactsList/index.jsx

```diff
@@ -134,24 +137,44 @@ function ContactsList() {
         ]}
       />
       <div className={containerClass}>
-        <table className={bem("table")}>
-          <tbody className={bem("table__body")}>
-            {contacts.map(({ name, avatar }) => (
-              <tr className={bem("table__body-row")}>
-                <td
-                  className={clsx(
-                    bem("table__body-cell"),
-                    bem("table__body-cell--align-end"),
-                    bem("table__body-cell--width-preview")
-                  )}
-                >
-                  <Avatar className={bem("avatar")} alt={name} src={avatar} />
-                </td>
-                <td>{name}</td>
-              </tr>
-            ))}
-          </tbody>
-        </table>
+        <Table
+          things={contacts}
+          className={bem("table")}
+          // prettier-ignore
+          ascIndicator={(
+            <span role="img" aria-label="Sorted in ascending order">
+              {" "}
+              <i className={clsx(bem("icon-caret-up"), bem("table__icon"))} />
+            </span>
+          )}
+          // prettier-ignore
+          descIndicator={(
+            <span role="img" aria-label="Sorted in descending order">
+              {" "}
+              <i className={clsx(bem("icon-caret-down"), bem("table__icon"))} />
+            </span>
+          )}
+        >
+          <TableColumn
+            property={hasPhotoPredicate}
+            header=""
+            datatype="url"
+            body={({ value }) => {
+              return (
+                <Avatar
```

Comment:
Note for later: we'll eventually want to replace `Avatar` with our own component that applies prism avatar classes to the react SDK `<Image>` component, which would allow us to make authenticated requests.

Reply:
Should we create a ticket for this as a reminder to ourselves?